components/eam/src/chemistry/modal_aero/modal_aero_convproc.F90

```diff
@@ -2477,59 +2478,76 @@ subroutine ma_resuspend_convproc(                           &
 !-----------------------------------------------------------------------
 ! arguments
 ! (note:  TMR = tracer mixing ratio)
-   integer,  intent(in)    :: pcnst_extd
+   integer,  intent(in)    :: pcnst_extd ! cloudborne aerosol dimension [unitless]
    real(r8), intent(inout) :: dcondt(pcnst_extd,pver)
-                              ! overall TMR tendency from convection
+                              ! overall TMR tendency from convection [kg/kg/s] 
    real(r8), intent(inout) :: dcondt_resusp(pcnst_extd,pver)
-                              ! portion of TMR tendency due to resuspension
+                              ! portion of TMR tendency due to resuspension [kg/kg/s]
                               ! (actually, due to the adjustments made here)
-   real(r8), intent(in)    :: const(pcnst_extd,pver)  ! TMRs before convection
 
-   real(r8), intent(in)    :: dp_i(pver) ! pressure thickness of level (in mb)
    integer,  intent(in)    :: ktop, kbot_prevap ! indices of top and bottom cloud levels ! REASTER 08/05/2015
 
 !-----------------------------------------------------------------------
 ! local variables
    integer  :: k, ll, la, lc, n
```

Comment:
Variables k, qdota, qdotc, and qdotac are not used here anymore. Would you please change n to nmode?